server/views/components/page-description/index.njk

```diff
@@ -3,7 +3,7 @@
   <h1 class="page-description__title">{{ title }}</h1>
   <span class="page-description__outro">
     {% if icon %}
-      {% component 'icon', {icon: icon, blockClass: 'page-description'} %}
+      {% icon icon %}
```

Comment:
I see what you mean. The icon will always be a clock, but the determining factor as to whether or not it displays is whether there's date information that we want to display with the article title. So the conditional would be better as

```
{% if outro %}
  {% icon icon %} {{ outro }}
{% endif %}
```

Better yet, I should probably change `outro` to `date` (or similar).